dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -354,7 +355,8 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
     // <deleted>.
     // Snowflake will refuse (CURRENT_TIMESTAMP - 168 hours) because it is beyond the
     // 7-day window allowed by the server-side function.
-    ZonedIntervalIterable intervals = ZonedIntervalIterable.forConnectorArguments(arguments);
+    ZonedIntervalIterable intervals =
```

Comment:
If I understand it correctly, this changes to interval to 1 day for all querylog/timeseries tasks, right?  Preferrably we want the interval to remain the same for all situations except for when running time series logs for --assessment tasks. i.e. we want to only change the default interval from HOUR to DAY only for time series.